src/components/GameMode.js

```diff
@@ -0,0 +1,26 @@
+export class GameMode {
+  constructor(config) {
+    this.config = config;
+  }
+
+  changeGameMode() {
+    let gameModeItems = document.querySelectorAll(`.menu__item`);
+    gameModeItems.forEach((modeItem) => {
+      modeItem.addEventListener(`click`, (e) => {
+        this.changeGameModeClassList(e), this.setSelectedGameMode();
+      });
+    });
+  }
+
+  changeGameModeClassList(e) {
+    let gameMode = document.querySelector('.menu__item--selected');
```

Comment:
const?

Reply:
poprawiłam chociaż ogólnie średnio mi się podoba trzymanie zmiennych w funkcjach, ale może tak ma być?

Reply:
to całkowicie normalne :)